app/lib/dttp/params/placement_assignment.rb

```diff
@@ -118,6 +121,12 @@ def funding_params
       def send_funding_to_dttp?
         FeatureService.enabled?(:show_funding) && FeatureService.enabled?(:send_funding_to_dttp)
       end
+
+      def academic_year
+        return ACADEMIC_YEAR_2020_2021 if trainee.course_start_date.between?(Time.parse("1/8/2020"), Time.parse("1/8/2021"))
```

Comment:
`between?` seems to be inclusive:
```
irb(main):004:0> Date.tomorrow.between?(Date.today, Date.tomorrow)
=> true
```
So don't we would want the end of these time ranges to all be "31/7/year" right?

Reply:
I had that originally but the test failed. Also, I hate dates.

Reply:
Ahh, its because you are doing `Time.parse` instead of  `Date.parse` which I guess isn't inclusive and is comparing them at the beginning of the day:
```
irb(main):016:0> Date.tomorrow.between?(Time.parse('5/8/2021'), Time.parse('6/8/2021'))
=> false
```